ui/k-info-panel/switch_info.kytos

```diff
@@ -36,23 +58,56 @@
                   'manufacturer': '',
                   'serial': '',
                   'hardware': '',
-                  'software': ''},
+                  'software': '',},
        interfaces: [],
        custom_properties: {},
        flows: [],
+       links: [],
+       headers: ['Links'],
      }
    },
    methods: {
      update_switch_content () {
        if(this.content === undefined) return
        this.interfaces = this.content.interfaces
+       
        this.custom_properties = this.content.custom_properties
        var self = this
        Object.keys(this.metadata).forEach(function (key) {
          let value = self.content[key]
          self.metadata[key] = String(value)
        });
        this.get_flows()
+       this.get_links()
+     },
+     get_links_success(data){
+       this.links = data['topology']['links']
+       
```

Comment:
You have to clear the table_link_body `this.table_link_body=[] ` or the array will always keep old values when you click another switch

Reply:
Now it is fixed.  Commit [24adf0a6946dcee7a18ed7afd21553a83195c085](https://github.com/kytos-ng/topology/pull/73/commits/24adf0a6946dcee7a18ed7afd21553a83195c085)